src/react/account/AccountHead.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+import { Head, HeadCenter, HeadLeft, HeadRight, HeadTitle, IconCircle} from '../ui-elements/ListLayout';
+import type { AppState } from '../../types/state';
+import { Button } from '@scality/core-ui';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+function AccountHead() {
```

Comment:
Can you add the IconCircle with the wallet when the user is in the empty state? The wallet color is textTertiary(#a7b6c3).

![image](https://user-images.githubusercontent.com/3807744/88964443-b0753a00-d2a9-11ea-97f9-2a0dd2c15f91.png)

Reply:
As discussed, I will leave it empty for now.